Formula/crowdin.rb

```diff
@@ -1,14 +1,10 @@
 class Crowdin < Formula
   desc "Command-line tool that allows to manage your resources with crowdin.com"
   homepage "https://support.crowdin.com/cli-tool/"
-  url "https://github.com/crowdin/crowdin-cli/releases/download/3.7.1/crowdin-cli.zip"
-  sha256 "5a80c5055d4d89dd72e7b308ad314c381db2a87454aef869f7be03cf96a06abd"
+  url "https://github.com/crowdin/crowdin-cli/releases/download/3.7.2/crowdin-cli.zip"
+  sha256 "ee9f838b819ccedc33c9b2537055e5ba7d7934561b24df1e1a6274cbd6e27f2d"
   license "MIT"
-
-  livecheck do
-    url "https://github.com/crowdin/crowdin-cli.git"
-    regex(/^v?(\d+(?:\.\d+)+)$/i)
-  end
+  head "https://github.com/crowdin/crowdin-cli.git", branch: "cli3"
```

Comment:
hmm, I dont think the head build would work, right?

Reply:
You're right, I was a bit too quick to change that. I'll revert it for now